File: Data Structures & Algorithms/is-anagram/submission-0.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        char mp[26] = {0};
        if(s.size() != t.size()) return false;
        for(auto c : s) mp[c - 'a']++;
        for(auto c : t) mp[c - 'a']--;
        for(auto n : mp)
            if(n != 0) return false;
        return true;
    }
};

File: Data Structures & Algorithms/is-palindrome/submission-0.py
class Solution:
    def isPalindrome(self, s: str) -> bool:
        cs = ""
        for i in range(len(s)):
            if s[i].isalnum():
                cs += s[i].lower()
        i=0
        j=len(cs) - 1
        while i < j:
            if cs[i] != cs[j]:
                return False
            i += 1
            j -= 1
        return True